docs/gateway/policies/apikey-authentication.md

```
@@ -340,6 +342,228 @@ curl -X POST "http://localhost:9090/apis/weather-api-v1.0/api-keys" \
+  "api_key": "apip_newvalue1234567890abcdef1234567890abcdef1234567890abcdef1234567890",
+  "expires_in": {
+    "duration": 60,
+    "unit": "days"
+  }
+}
+```
+
+**Request Body Schema:**
+
+| Field | Type | Required | Description |
+|-------|------|----------|-------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)